test/e2e/apps/statefulset.go

```diff
@@ -860,14 +862,33 @@ var _ = SIGDescribe("StatefulSet", func() {
 			if err != nil {
 				framework.Failf("Failed to put scale subresource: %v", err)
 			}
-			framework.ExpectEqual(scaleResult.Spec.Replicas, int32(2))
+			framework.ExpectEqual(scaleResult.Spec.Replicas, int32(2), "")
```

Comment:
Ditto.

Reply:
The extra `"  " ` was an automatic `gofmt` change that I can remove.